Vagrantfile

```diff
@@ -30,10 +39,20 @@ Vagrant.configure("2") do |config|
   # Mounting with uid and gid is necessary as the provision script will create
   # this user, so mounting before does not work with an owner specified by name.
   # Also, provision needs to use vagrant as ssh user (since evap does not exist yet)
-  config.vm.synced_folder ".", "/evap", mount_options: ["uid=1042", "gid=1042"]
-  if ARGV[0] == "ssh" or ARGV[0] == "ssh-config"
-    config.ssh.username = 'evap'
+
+  config.vm.provider :docker do |d, override|
+    override.vm.synced_folder ".", "/evap", docker_consistency: "cached"
+
+    override.vm.provision "shell", path: "deployment/provision_vagrant_vm.sh", args: "docker"
+    override.vm.provision "shell", inline: "service redis-server start && service postgresql start", run: "always", privileged: true
```

Comment:
This runs when _provisioning_, but shouldn't it run every time the container is booted? I suppose there should be an equivalent to `systemctl enable`, probably `service enable` or so. You can also run this in the script, this is what we expect in both setups anyway

Reply:
it does run always, i think. this is only needed if docker is used. I had some issues with it, this is the best and simplest I was able to find.